sunpy/timeseries/timeseriesbase.py

```diff
@@ -334,6 +335,10 @@ def concatenate(self, otherts, **kwargs):
         at once.
         """
 
+        # check to see if nothing needs to be done
+        if self == otherts:
```

Comment:
Yes, @Alex-Ian-Hamilton defined it earlier. It mostly just checks if the data is the same.